<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerProfileControl</class>
 <widget class="QWidget" name="PowerProfileControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Power Profiles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_PPQuiet">
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 170, 255);</string>
          </property>
          <property name="text">
           <string>QUIET</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_PPBalanced">
          <property name="styleSheet">
           <string notr="true">color: rgb(145, 145, 145);</string>
          </property>
          <property name="text">
           <string>BALANCED</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_PPPerformance">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>PERFORMANCE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_PPCustom">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 85, 255);</string>
          </property>
          <property name="text">
           <string>CUSTOM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
